pyaedt/desktop.py

```diff
@@ -363,203 +366,193 @@ def current_version_student(self):
+            StandalonePyScriptWrapper.CreateObjectNew(non_graphical)
+        else:
+            StandalonePyScriptWrapper.CreateObject(version)
+        if non_graphical:
+            os.environ['PYAEDT_DESKTOP_LOGS'] = 'False'
+        processID2 = []
+        if IsWindows:
+            processID2 = self._get_tasks_list_windows(student_version)
+        proc = [i for i in processID2 if i not in processID]
+        if not proc:
+            proc = processID2
+        if len(processID2) > 1:
+            if non_graphical:
+                self._main.close_on_exit = False
+            else:
+                self._main.close_on_exit = False
+                self._dispatch_win32(version)
+        elif version_key >= "2021.1":
+            self._main.close_on_exit = True
+            if student_version:
+                print("Info: {} Student Started with Process ID {}".format(version, proc[0]))
+            else:
+                print("Info: {} Started with Process ID {}".format(version, proc[0]))
+            context = pythoncom.CreateBindCtx(0)
+            running_coms = pythoncom.GetRunningObjectTable()
+            monikiers = running_coms.EnumRunning()
+            for monikier in monikiers:
+                m = re.search(version[10:] + r"\.\d:" + str(proc[0]), monikier.GetDisplayName(context, monikier))
+                if m:
+                    obj = running_coms.GetObject(monikier)
+                    self._main.isoutsideDesktop = True
+                    self._main.oDesktop = win32com.client.Dispatch(obj.QueryInterface(pythoncom.IID_IDispatch))
+                    break
+        else:
+            warnings.warn(
+                "PyAEDT is not supported in AEDT versions older than 2021.1. Trying to launch it with PyWin32.")
+            self._dispatch_win32(version)
+
+    def _init_logger(self):
+        # Set up the log file in the AEDT project directory
+        self.logger = logging.getLogger(__name__)
+        if not self.logger.handlers:
+            if "oDesktop" in dir(self._main):
+                project_dir = self._main.oDesktop.GetProjectDirectory()
+            else:
+                if os.name == "posix":
+                    project_dir = os.environ["TMPDIR"]
+                else:
+                    project_dir = os.environ["TEMP"]
+            self.logfile = os.path.join(project_dir,
+                                        "pyaedt{}.log".format(datetime.datetime.now().strftime("%Y%m%d_%H%M%S")))
+            logging.basicConfig(filename=self.logfile, format='%(asctime)s:%(name)s:%(levelname)-8s:%(message)s',
+                                level=logging.DEBUG, datefmt='%Y/%m/%d %H.%M.%S', filemode='w')
+        return True
+
     def __init__(self, specified_version=None, NG=False, AlwaysNew=True, release_on_exit=True, student_version=False):
         """Initialize desktop."""
-        self._main = sys.modules["__main__"]
+        self._main = sys.modules['__main__']
```

Comment:
Is there any reason for this?  Current standard via `black` is to use double quotes.  We don't document this anywhere and we should.  Pinging @jleonatti

Reply:
```suggestion
        self._main = sys.modules["__main__"]
```